### ISSUE TITLE
Create a custom PHPMD ruleset.

### ISSUE THREAD

**Issue by bmitch** [bmitch/churn-php#129]
Instead of relying on

```
vendor/bin/phpmd src text codesize,unusedcode,naming
```

Would be nice to have our own ruleset for when we want to adjust settings, like the number of characters allowed in a variable.

https://phpmd.org/documentation/creating-a-ruleset.html

Once this is done we can expand on the variable names in `ChurnCommand->getDirectoriesToScan()`
